lib/connect.js

```diff
@@ -27,29 +24,45 @@ module.exports = function connect (dht, publicKey, opts = {}) {
     handshake: (opts.createHandshake || defaultCreateHandshake)(keyPair, publicKey),
     request: null,
     requesting: false,
-    protocols: PROTOCOL.TCP | PROTOCOL.UTP,
+    protocols: PROTOCOL.UDX,
```

Comment:
@mafintosh Do we still need this?

Reply:
no, but let's save this for a "handshake bikeshed session 2022 edition"